manage.py

```diff
@@ -106,7 +106,7 @@ def run_worker():
         password=app.config['RQ_DEFAULT_PASSWORD'])
 
     with Connection(conn):
-        worker = Worker(map(Queue, listen))
+        worker = Worker(list(map(Queue, listen)))
```

Comment:
I believe in this case that a list may cause the Worker to file (I'll take a look at it tonight).